release-lockfile.spec.md

```diff
@@ -102,7 +102,7 @@ A deployed instance of the `Wallet` contract would be of of type `Wallet`.
 
 The name found in the source code which defines a specific *contract type*.
 These names **must** conform to the regular expression
-`[a-zA-Z][-a-zA-Z0-9_]*`.
+`[a-zA-Z][-a-zA-Z0-9_]{0,255}`
```

Comment:
I chose 256 as an arbitrary upper limit on the allowed length.  Seems reasonable.

Reply:
👍